src/main/java/org/tron/core/net/node/NodeImpl.java

```diff
@@ -630,6 +630,11 @@ public synchronized void disconnectInactive() {
 
 
   private void onHandleInventoryMessage(PeerConnection peer, InventoryMessage msg) {
+    int count = peer.getNodeStatistics().messageStatistics.tronInTrxInventoryElement.getCount(10);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Assign this magic number 10 to a well-named constant, and use the constant instead. ([squid:S109](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS109))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1672&issues=AWbTi2UkLpqqSUlcbPrr&open=AWbTi2UkLpqqSUlcbPrr)

[ISSUE_KEY]: AWbTi2UkLpqqSUlcbPrr